docs/2.8.7/versions.json

```diff
@@ -0,0 +1,6 @@
+{
+  "daml": "2.8.6",
```

Comment:
This should be `2.8.7` for daml and canton, no?

Reply:
No, component versions here. We have a Daml Enterprise 2.8.7, but sdk, canton, and drivers are all still at 2.8.6.